elements/pf-chip/pf-chip-group.css

```diff
@@ -117,7 +118,7 @@
   display: inline-flex;
   align-items: center;
   justify-content: center;
-  min-height: 44px;
+  min-height: var(--_pf-chip-group-target-size);
```

Comment:
@nikkimk Please revert. We can't differ from pfv4, even if upstream styles are broken

Raise it as an issue on patternfly

Reply:
I reverted this in a820b0. it's not great that we have to ship something broken, but this project needs to replicate the upstream behaviour, and the proposed change diverges too much from the patternfly designs. 

in projects downstream from here we can fix it, and this should be raised as an issue upstream.